### PR TITLE
Disable T25 test case for interop testing

### PR DESCRIPTION
**Pull request by skondkar**
PR to disable T25 test case for interop testing. Reference bug - https://issues.redhat.com/browse/OSSM-1505

### CODE REVIEW

File: tests/test_cases.go [hunk 1]
```diff
@@ -194,10 +194,10 @@ var testCases = []testing.InternalTest{
 		F:    authorizaton.TestTrustDomainMigration,
 	},
 
-	testing.InternalTest{
-		Name: "T25",
-		F:    ossm.TestExtensionInstall,
-	},
+	//testing.InternalTest{
```
Comment (jwendell) [inline tests/test_cases.go:197]: Why not disabling it only on OCP 4.11?

**Comment by mattmahoneyrh** [comment]
@skondkar Consider checking for OCP 4.11 and greater, as the test will still run and fail for OCP 4.12 (for example).

**Comment by jwendell** [comment]
> @skondkar Consider checking for OCP 4.11 and greater, as the test will still run and fail for OCP 4.12 (for example).

I thought about this too, but I think it's a good idea to check for 4.11 only. It might be that 4.12 has the issue fixed.

**Comment by jwendell** [comment]
Yes, that's why I think it's better not to ignore > 4.11. If we do that, and the bug is fixed in 4.12, we would not notice it because the test is disabled. IMO it's better to have it fail for 4.12 (so we can disable it again) than not run it by mistake.

**Comment by mattmahoneyrh** [comment]
test on .12 before this PR is merged and update PR to disable on .12 if needed OR do so after this PR is merged and then if still fails on .12 create a new PR. Either way is fine with me.

**Comment by tsze-redhat** [comment]
We need the test to be disabled for 4.12 (until it is fixed) so that interop test can pass.

**Comment by FilipB** [comment]
Added task to implement ideal solution for this situation - https://issues.redhat.com/browse/OSSM-1904
Use checker which is able to check jira/bz status and skip/run given test based on that status.

**Comment by FilipB** [comment]
Also we should cherry pick this change to maistra-2.2

**Comment by FilipB** [comment]
I will merge it now. We can create another PR for OCP 4.12 later

**Pull request merged**